build/asset-manifest.json

```diff
@@ -20,19 +20,19 @@
     "static/media/email.svg": "/static/media/email.svg",
     "static/media/user-managment.svg": "/static/media/user-managment.svg",
     "static/media/lock.svg": "/static/media/lock.svg",
+    "static/media/provider-apple.svg": "/static/media/provider-apple.svg",
     "static/media/checkmark-green.svg": "/static/media/checkmark-green.svg",
     "static/media/plus.svg": "/static/media/plus.svg",
     "static/media/green-check.svg": "/static/media/green-check.svg",
     "static/media/clear.svg": "/static/media/clear.svg",
```

Comment:
change it to use the broweser default for drop downs.. don't use our own custom drop down!

Reply:
done